admin/spec/service/data-access/pupil.data.service.spec.js

```diff
@@ -117,21 +117,27 @@ describe('pupil.data.service', () => {
       expect(mockPupil.verify()).toBe(true)
     })
   })
-  describe('#saveMultiple', () => {
+  describe('#updateMultiple', () => {
```

Comment:
what's the hash for?

Reply:
it's just a pattern we are using to signify that's a method.
Happy to remove it if it's confusing